src/Compilers/CSharp/Portable/Lowering/Instrumentation/ModuleCancellationInstrumenter.cs

```diff
@@ -198,6 +198,7 @@ public override BoundStatement InstrumentGotoStatement(BoundGotoStatement origin
                         parametersWithCancellationToken.AsSpan(0, methodDefinition.Parameters.Length),
                         typeMap,
                         MemberSignatureComparer.RefKindCompareMode.ConsiderDifferences,
+                        considerDefaultValues: true,
```

Comment:
>considerDefaultValues: true, [](http://example.com/codeflow?start=24&length=28)

This looks suspicious. First, I do not think we want to change behavior of this code path because it is outside of our goal for this PR (fix the shape of inferred anonymous delegate). Second, we probably do not want to change current behavior in general because at this point we have all arguments and default values don't really matter for what we are going to pass.